.npmignore

```diff
@@ -13,3 +13,4 @@
 /build
 /builds
 /dist/tsbuildinfo
+/images
```

Comment:
Scratch this. I don't think it should be done.

Reply:
👍🏻